pkg/skaffold/build/builder_mux.go

```diff
@@ -106,7 +108,8 @@ func (b *BuilderMux) Build(ctx context.Context, out io.Writer, tags tag.ImageTag
 		}
 		var built string
 
-		if platforms.IsMultiPlatform() && !SupportsMultiPlatformBuild(*artifact) {
+		// buildx creates multiplatform images via buildkit directly
+		if platforms.IsMultiPlatform() && !SupportsMultiPlatformBuild(*artifact) && !b.buildx {
 			built, err = CreateMultiPlatformImage(ctx, out, artifact, tag, platforms, artifactBuilder)
```

Comment:
I appreciate the simplicity of this approach, and this is really a curiosity more than a comment. Is there a reason not to instead extend CreateMultiPlatformImage to run individual buildx instances for each platform, and then stitch them into a single manfiest using the built-in skaffold stitching?

In particular, in my use case, I would want to run (via GH actions) separate /native/ builders (our build is too slow on arm to want to use qemu), define them as multiple remotes, and then stitch them in the main skaffold build command.

Reply:
Using the native buildkit multi-platform can offload the build parallelization and image stitching from skaffold, and it would support more strategies: 
https://github.com/docker/buildx?tab=readme-ov-file#building-multi-platform-images

> 1. Using the QEMU emulation support in the kernel
> 2. Building on multiple native nodes using the same builder instance
> 3. Using a stage in Dockerfile to cross-compile to different architectures

I addressed more about using native nodes in my other reply, I think it should be possible and match your use case without needing separate builds or stitching: https://github.com/GoogleContainerTools/skaffold/pull/9648#issuecomment-2810512129

Reply:
That makes great sense to me, thanks for explaining, and its really cool that buildx has support for "fanning out" to native builders built-in. Somehow I had never noticed that despite using buildx for a couple years.